src/deploy/firestore/release.ts

```diff
@@ -0,0 +1,19 @@
+import { get } from "lodash";
```

Comment:
mild preference for `import * as _ from "lodash"`

Reply:
sure.